omnipaxos/src/sequence_paxos/mod.rs

```diff
@@ -409,11 +402,13 @@ where
     }
 
     fn accept_stopsign(&mut self, ss: StopSign) {
+        let accepted_idx = self.internal_storage.get_accepted_idx();
         self.internal_storage
-            .set_stopsign(StopSignEntry::with(ss, false))
+            .set_stopsign(StopSignEntry::with(ss, accepted_idx))
             .expect("storage error while trying to write stopsign");
         if self.state.0 == Role::Leader {
-            self.leader_state.set_accepted_stopsign(self.pid);
+            self.leader_state
+                .set_accepted_idx(self.pid, accepted_idx + 1);
```

Comment:
Let's unify the access to the `accepted_idx`  by always calling `self.internal_storage.get_accepted_idx()`
```suggestion
            let accepted_idx = self.internal_storage.get_accepted_idx();
            assert_eq!(accepted_idx, ss_log_idx + 1);
            self.leader_state
                .set_accepted_idx(self.pid, accepted_idx);
```